File: client/src/app/_models/admin/orderAssessment.ts
import { IOrderAssessmentItem } from "./orderAssessmentItem";


export interface IOrderAssessment {
     id: number;
     orderAssessmentId: number;
     customerName: string;
     orderId: number;
     orderNo: number;
     dateDesigned: Date;
     orderAssessmentItems: IOrderAssessmentItem[];
}

export interface IOrderAssessmentDto {
     id: number;
     orderAssessmentId: number;
     customerName: string;
     orderId: number;
     orderNo: number;
     dateDesigned: Date;
}

File: api/Entities/Admin/Order/OrderAssessment.cs
namespace api.Entities.Admin.Order
{
    public class OrderAssessment: BaseEntity
    {
        public int OrderId { get; set; }
        public int OrderNo { get; set; }
        public DateTime OrderDate { get; set; }
        public string CustomerName { get; set; }
        public string DesignedByUsername { get; set; }
        public DateTime DateDesigned {get; set;}
        public ICollection<OrderAssessmentItem> OrderAssessmentItems { get; set; }
    }
}